test/stress/child.js

```diff
@@ -0,0 +1,87 @@
+const https = require('https')
+const path = require('path')
+const pify = require('pify')
+const fs = pify(require('fs'))
+const uuid = require('uuid')
+const _ = require('lodash/fp')
+const { PerformanceObserver, performance } = require('perf_hooks')
+
+const utils = require('./utils')
+const variables = require('./utils/variables')
+
+var certificate = {}
+var connectionInfo = {}
+
+async function getCert (machineIndex) {
+  const key = fs.readFile(path.resolve(__dirname, 'machines', `${machineIndex}`, 'client.key'))
+  const cert = fs.readFile(path.resolve(__dirname, 'machines', `${machineIndex}`, 'client.pem'))
+
+  return Promise.all([key, cert]).then(values => {
+    return { key: values[0], cert: values[1] }
+  }).catch(err => {
+    console.err('The following error when reading the certificate: ', err)
+    return null
+  })
+}
+
+async function getConnectionInfo (machineIndex) {
```

Comment:
no need for async
also I think you can do this:
```suggestion
const getConnectionInfo = (machineIndex) => fs.readFile(path.resolve(__dirname, 'machines', `${machineIndex}`, 'connection_info.json'))
```